packages/foreman/foreman/foreman.spec

```diff
@@ -98,8 +98,6 @@ Requires: %{?scl_prefix}rubygem(rails-i18n) >= 5.0
 Requires: %{?scl_prefix}rubygem(rails-i18n) < 6.0
 Requires: %{?scl_prefix_ror}rubygem(i18n) >= 1.1
 Requires: %{?scl_prefix_ror}rubygem(i18n) < 2.0
-Requires: %{?scl_prefix_ror}rubygem(turbolinks) >= 2.5.4
```

Comment:
should we also obsolete this?

Reply:
No, obsoleting something from a different collection isn't correct IMHO.